### ISSUE TITLE
View / Glyph Sequence does not scroll on Hi-Def Screen

### ISSUE THREAD

**Issue by Windsurfer01** [johanmattssonm/birdfont#165]
It may be because I use Linux Mint, based on Ubuntu. When I enter a sequence of characters like oeabgpdq, individual characters have to be selected via the OverView tab. It is also not possible to scroll along the glyph sequence, because there is no scroll bar.

**Comment by johanmattssonm** [comment]
Is this in the kerning / spacing tab?

**Comment by Windsurfer01** [comment]
Yes it is. There are two attachments: the full screen showing the Overview tab plus previously selected glyphs, and a close up of the Overview tab. Clicking on any of them either selects them from the Glyph Sequence, or places them into it. Whatever has been selected has focus for editing. It works, but cannot be scrolled on my PC. 
![Fullscreen](https://github.com/johanmattssonm/birdfont/assets/3854941/0c504064-be93-4de7-bc0e-26aebc5e9d31)
![Overview Tab](https://github.com/johanmattssonm/birdfont/assets/3854941/64c7fe06-3428-40be-b499-0c68c53fc78a)

**Comment by johanmattssonm** [comment]
Thanks. I will try to install Mint on a usb stick later tonight. Does keyboard input work in other parts of the program?

**Comment by johanmattssonm** [comment]
Which desktop environment should I use? Cinnamon, MATE or Xfce?

**Comment by johanmattssonm** [comment]
I tested the flatpak on Mint Cinnamon 21.3 things did work. I just realized that the screenshots you added are from the glyph editing view and not the kerning tab; you can press space and click to more around on the canvas. The glyphs around your letter can't be clicked on but I want to fix that. Maybe in version 6.1, I cross my fingers and hope that it will be done soon.

**Comment by Windsurfer01** [comment]
Sorry, I did not think to mention that I use Cinnamon. It is usable as it is, but not as expected.
I have tried the space bar and clicking. But, it does not work for me.
I see that you have been developing BirdFont for many years. Any comments that I make are in the spirit of helpfulness, and I'll let you know of any difficulties I have. It took me a long time to discover most of FontForge, and how to use it. Such complex tools are not going to be learnt in five minutes.

**Comment by johanmattssonm** [comment]
I do appreciate your feedback, it is very constructive.

I am still scratching my head regarding the scrolling issue. Can you do a short screen cast?

**Comment by Windsurfer01** [comment]
I'll send two videos via WeTransfer.

**Comment by johanmattssonm** [comment]
Many thanks. I think this explains it, I clearly have some work todo. I think you can move around in the editing view with the hand tool.
